plugins/woocommerce/client/admin/client/settings-payments/components/field-placeholder/field-placeholder.tsx

```diff
@@ -0,0 +1,27 @@
+/**
+ * External dependencies
+ */
+import { Placeholder } from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import './field-placeholder.scss';
+
+export type FieldPlaceholderSize = 'small' | 'medium' | 'large';
+
+export const FieldPlaceholder = ( {
```

Comment:
```suggestion
export const PaymentGatewayFieldPlaceholder = ( {
```

Reply:
This can be used generically for any field, so IMO `FieldPlaceholder` is an appropriate name for it.

Reply:
OK, agree 🙂